Northstar.Client/mod/resource/northstar_client_localisation_mspanish.txt

```diff
@@ -0,0 +1,311 @@
+		"fp_embark_enabled" "Embarques/Ejecucciones en primera persona"
+		"classic_rodeo" "Rodeo Clásico"
+		"oob_timer_enabled" "Temporizador de limite de mapa"
+		"riff_instagib" "Modo Instagib"
+
+		"riff_player_bleedout" "Desangrado de piloto"
+		"player_bleedout_forceHolster" "Enfundar armas mientras estas derribado"
+		"player_bleedout_forceDeathOnTeamBleedout" "Morir al estar todos derribados"
+		"player_bleedout_bleedoutTime" "Tiempo de desangrado"
+		"player_bleedout_firstAidTime" "Tiempo de primeros auxilios"
+		"player_bleedout_firstAidTimeSelf" "Tiempo de auto resucitación"
+		"player_bleedout_firstAidHealPercent" "Porcentaje de salud de primeros auxilios"
+		"player_bleedout_aiBleedingPlayerMissChance" "Probabilidad de fallo de IAs derribadas"
+
+		// coop stuff
+		"PL_sp_coop" "(Sin terminar) Campaña cooperativa"
+		"PL_sp_coop_lobby" "Lobby Campaña cooperativa"
+		"PL_sp_coop_desc" "Juega a traves de la campaña junto con amigos"
+		"PL_sp_coop_hint" "Juega a traves de la campaña junto con amigos"
+		"PL_sp_coop_abbr" "SP"
+
+		"SP_TRAINING" "El circuito del piloto"
+		"SP_TRAINING_CLASSIC_DESC" "Simulación de entrenamiento del Capitán Lastimosa."
+
+		"SP_CRASHSITE" "BT-7274"
+		"SP_CRASHSITE_CLASSIC_DESC" "Jack Cooper conoce a BT-7274."
+
+		"SP_SEWERS1" "Sangre y óxido"
+		"SP_SEWERS1_CLASSIC_DESC" "Cooper y BT emprenden el viaje para reunirse con el comandante Anderson."
+
+		"SP_BOOMTOWN_START" "Viaje al abismo"
+		"SP_BOOMTOWN_START_CLASSIC_DESC" "Enfréntate a consecuencias inesperadas al pasar por un atajo subterráneo."
+
+		"SP_HUB_TIMESHIFT" "Causa y efecto"
+		"SP_HUB_TIMESHIFT_CLASSIC_DESC" "Se descubrió un fenómeno extraño en las coordenadas del comandante Anderson."
+
+		"SP_BEACON" "La baliza"
+		"SP_BEACON_CLASSIC_DESC" "Cooper y BT intentan comunicar los planes de la IMC a la flota que queda."
+
+		"SP_TDAY" "Prueba de fuego"
+		"SP_TDAY_CLASSIC_DESC" "Las habilidades del titán de Cooper se ponen a prueba en una gran batalla para capturar el Arca."
+
+		"SP_S2S" "El Arca"
+		"SP_S2S_CLASSIC_DESC" "Cooper y BT van de nave en nave persiguiendo el Arca."
+
+		"SP_SKYWAY_V1" "El Arma moldeadora"
+		"SP_SKYWAY_V1_CLASSIC_DESC" "Kuben Blisk captura a BT y Cooper."
+
+		// Better.Serverbrowser
+		"SERVERS_COLUMN" "Sevidores"
+		"PLAYERS_COLUMN" "Jugadores"
+		"MAP_COLUMN" "Mapa"
+		"GAMEMODE_COLUMN" "Modo de juego"
+		"LATENCY_COLUMN" "Latencia"
+		"SEARCHBAR_LABEL" "Buscar:"
+		"MAP_FILTER" "Mapa"
+		"GAMEMODE_FILTER" "Modo de juego"
+		"HIDE_FULL_FILTER" "Ocultar servidores llenos"
+		"HIDE_EMPTY_FILTER" "Ocultar servidores vacios"
+		"HIDE_PROT_FILTER" "Ocultar servidores protegidos"
+		"SERVER_DESCRIPTION" "Descripcion"
+		"SERVER_MODS" "Mods"
+		"CLEAR_FILTERS" "LIMPIAR"
+		"JOIN_BUTTON" "UNIR"
+
+		"SWITCH_YES" "Si"
+		"SWITCH_NO" "No"
+		"SWITCH_ANY" "Cualquiera"
+
+		"CONNECTING" "Conectando..."
+		"INGAME_PLAYERS" "Jugadores: ^6BA6C400%s1"
+		"TOTAL_SERVERS" "Servidores: ^C46C6C00%s1"
+
+
```

Comment:
```suggestion

```
Remove this whitespace so it matches the number of lines we have in english.txt

Reply:
Done